scout/adapter/mongo/panel.py

```diff
@@ -187,7 +188,10 @@ def panel(self, panel_id):
             dict: panel object or `None` if panel not found
         """
         if not isinstance(panel_id, ObjectId):
-            panel_id = ObjectId(panel_id)
+            try:
```

Comment:
👍🏻

Reply:
Well found with the missing column etc! Let's keep it simple and merge this and fix it in a separate PR.